src/components/DataTable/DataTableRow/DataTableRow.tsx

```diff
@@ -41,10 +41,19 @@ const DataTableRowPresentation = (props: DataTableRowProps) => {
 
     const result = useMemo(
         () =>
-            columns.map((item, i) => (
-                <Fragment key={i}>{renderCellComponent({ item, index: i })}</Fragment>
+            columns.map((item, i, self) => (
+                <DataTableCellContext.Provider
+                    value={{
+                        column: item,
+                        columnIndex: i,
+                        row: rowId,
+                        rowIndex: index,
+                        isLast: self.length - 1 === i,
+                    }}>
+                    <Fragment key={i}>{renderCellComponent({ item, index: i })}</Fragment>
```

Comment:
Context provider is not need here. isLast can be passed down as a prop to renderCellComponent

Reply:
I didn't want to prop drill, so i moved the context outside, like it was done with DataHeaderCell

Reply:
This is not prop drilling. You're passing down other props anyways.

Reply:
Plus, inline object above will make every cell rerender whenever the row update. To avoid it, you'll have to create a component and pass this down as prop anyways

Reply:
it's in a useMemo